default.nix

```diff
@@ -15,4 +15,5 @@
   overlays = import ./overlays; # nixpkgs overlays
 
   adr-tools = pkgs.callPackage ./pkgs/adr-tools { };
+  elm-graphql = pkgs.callPackage ./pkgs/elm-graphql { };
```

Comment:
As a staging space before pushing expressions up to the nixpkgs repo, it'll be make it easier to make upstream PRs if we use [the same structure as nixpkgs's `nodePackages`](https://github.com/NixOS/nixpkgs/blob/49fb21232ff08a2f591237a97ebbbd7beda8d9c2/pkgs/top-level/all-packages.nix#L3955-L3957). According to [the docs](https://github.com/NixOS/nixpkgs/blob/49fb21232ff08a2f591237a97ebbbd7beda8d9c2/doc/languages-frameworks/node.section.md), `pkgs.nodePackages` contain end user software packages, which matches what we're trying to extract with this PR.